src/DisplayBoard.cpp

```diff
@@ -26,10 +26,10 @@ namespace DisplayBoard
 {
 std::string display(const Board& b)
 {
-    auto width = b.getRowsNumber();
-    auto height = b.getColsNumber();
-    auto widthRows = b.getLongestRowLenght();
-    auto heightCol = b.getLongestColLenght();
+    auto width = b.getRowSize();
+    auto height = b.getColSize();
+    auto widthRows = b.getLongestCluesLenghtInRows();
```

Comment:
What about getWitdhOfClues() and getHeightOfClues()? 
I am asking also other team members ;)

Reply:
we never referred to those values in terms of height or width, even in terms of size of board. I would stock to this version because it directly says what is returned from function.

Reply:
I agree with jzych.